### DISCUSSION TITLE
How can i change the size of the DatePicker Calender?

### DISCUSSION THREAD

**Discussion by FardinFarnezhad** [HandyOrg/HandyControl#761] [Q&A] [answered]
hey i am using the Persian DatePicker but the numbers are a bit small so i want to make them bigger i tried using viewbox and changing the font size but they didn't work so any suggestions?

**Comment by ghost1372** [comment, marked as answer]
viewbox work you dont need to change font size, you just need to play with **Stretch**, **width** and **height**
![image](https://user-images.githubusercontent.com/9213496/113470525-8e047380-946b-11eb-8b66-ab576cd5ff29.png)
